### PR TITLE
Pba 3384 Ad Overlay UI

### PR DESCRIPTION
**Pull request by Matthew-C**

### CODE REVIEW

**Comment by lental** [comment]
when do the overlays dismiss? ever?

**Comment by lental** [comment]
fyi - i'm fine if this is 80% functionality, but i do want to know what we're missing - I think PBI has a parameter for showing the "x" dismiss button, and not, and i thought overlays had timeouts. are either of these true?

**Comment by Matthew-C** [comment]
the overlays are dismissed on next linear ad.

**Comment by Matthew-C** [comment]
Yes PBI has a configurable x dismiss button. I don't think they have timeouts (VAST spec has no way to specify a timeout). I can add the x dismiss button.

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java [hunk 1]
```diff
@@ -231,6 +232,20 @@ public static WritableMap buildAdsParams(Object data) {
     return params;
   }
 
+  public static WritableMap buildAdOverylayParams(AdOverlayInfo overlayInfo) {
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java:235]: typo on Ad Overlay

File: sdk/react/panels/videoView.js [hunk 2]
```diff
@@ -155,11 +156,12 @@ var VideoView = React.createClass({
       }} />);
   },
 
-  _renderPlaceholder: function() {
+  _renderPlaceholder: function(adOverlay) {
```
Comment (lental) [inline sdk/react/panels/videoView.js:159]: Does it necessarily have to be part of the placeholder? can you just add this below the overlay?

File: sdk/react/panels/videoView.js [hunk 3]
```diff
@@ -256,6 +258,39 @@ var VideoView = React.createClass({
...
+    var height = this.props.adOverlay.height;
+    if (width > this.props.width) {
+      height = width / this.props.width * height;
+      width = this.prop.width;
+    }
+    var left = (this.props.width - width) / 2;
+
+    return (
+      <TouchableHighlight 
+        style={{position: "absolute", left: left, bottom: 10, width:width, height:height}}
```
Comment (lental) [inline sdk/react/panels/videoView.js:276]: does style need to be absolute?  can you do this as simnple flex?

**Pull request closed without merging**